src/index.ts

```diff
@@ -141,5 +144,35 @@ export default function (listener: FlatfileListener) {
     configureSharedUses({ listener, apiService: HcmShowApiService });
   });
 
+  listener.namespace('space:hcmembedded', (listener) => {
+    listener.use(hcmEmbeddedSpaceConfigure);
+    listener.use(hcmPrefillData());
+    configureSharedUses({ listener, apiService: HcmShowApiService });
+  });
+
+  listener.namespace('space:hcmfilefeed', (listener) => {
+    listener.use(hcmFileFeedSpaceConfigure);
+    configureSharedUses({ listener, apiService: HcmShowApiService });
+
+    listener.use(
+      filefeedAutomap({
+        apiService: HcmShowApiService,
+        matchFilename: /^benefits.*$/i,
+        defaultTargetSheet: BENEFITS_SHEET_NAME,
+      })
+    );
+
+    listener.on('**', (event) => {
+      // Send certain filefeed events to hcm.show
+      if (
+        event.topic.includes('records:') ||
+        (event.topic === 'job:completed' &&
+          event?.payload?.status === 'complete')
+      ) {
+        HcmShowApiService.sendFilefeedEvent(event);
+      }
+    });
+  });
```

Comment:
As a future TODO I think this filefeed hook could be abstratced into a use function